ui/app/src/components/SystemStatus/GlobalDialogManager.tsx

```diff
@@ -211,6 +211,7 @@ function GlobalDialogManager() {
       <HistoryDialog
         open={state.history.open}
         versionsBranch={versionsBranch}
+        permissions={permissions}
```

Comment:
Should/could we send the permissions for the item in question instead of the whole thing?